components/molecules/ContributorListTableRow/contributor-list-table-row.tsx

```diff
@@ -134,13 +168,11 @@ const ContributorListTableRow = ({
 
           <div className="flex items-center justify-between py-3 border-b">
             <div>Languages</div>
-            {contributorLanguageList.length > 0 ? (
+            {contributorLanguageList && (
               <p>
-                {contributorLanguageList[0]}
-                {contributorLanguageList.length > 1 ? `,+${contributorLanguageList.length - 1}` : ""}
+                {contributorLanguageList[0] && getLanguageAbbreviation(contributorLanguageList[0])}
```

Comment:
refactor (non-blocking): Instead of referencing `contributorLanguageList[0]` and `contributorLanguageList[1]` multiple times, they could be destructured just after line 109 before the JSX is returned like so.

```typescript
const [firstLanguage, secondLanguage] = contributorLanguageList;
```

and then you can check if they exist or not, e.g.

```jsx
<p>
  {firstLanguage && getLanguageAbbreviation(firstLanguage)}
  {secondLanguage && getLanguageAbbreviation(secondLanguage)}
</p>
````

Reply:
That makes sense. Made the change 👍